lib/AnyEvent/HTTPD/Router.pm

```diff
@@ -47,14 +50,36 @@ sub dispatcher { shift->{dispatcher} }
 sub reg_routes {
     my $self = shift;
 
-    croak 'arguemnts to reg_routes are required' if @_ == 0;
+    croak 'arguments to reg_routes are required' if @_ == 0;
     croak 'arguments to reg_routes are confusing' if @_ % 3 != 0;
+    my $route_class = $self->{route_class};
```

Comment:
$route_class is from a old version of this module and not needed, could you remove line 55